GSA/digitalgov.gov
# Huskey Precommit Script Update

## Summary

This PR will update the Huskey precommit command to run the `lint:fix` command.  This will cut down on running multiple commands before a commit.  

The benefit of this is, that the files will still be flagged for review but files will appear in the source control.  There have been times that files that need some content changes such as spacing or lists require you to abort the commit and then run the fix command.  Now this will all be done and any files that need fixing will be separate and easy to commit with the update of the lining fix. 

The goal is to streamline the linting/prettier process for committing work. 

Below is the view that you will have after you commit any work that you have done and files are fixed with the `lint:fix` command
<img width="343" alt="Screenshot 2023-11-17 at 12 17 13 PM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/c4ab85f3-be79-4dcc-8e4f-ce46fa5c71e5">

### Preview

No Preview needed... simple script update.



### Solution

added the `:fix` flag to the lining commands 

### How To Test

1. review the files to ensure that `:fix` was added to the command 
2. branch off of this working branch and change a file by adding a space or some small edit and see that after you commit that up the fix is run and you have the option to then stage that new file. 

---

### Dev Checklist
